src/app/learn/get-started/advanced/accessibility-unit-tests/index.html

```diff
@@ -32,70 +32,41 @@
 
   <p>
-    After you import the <sky-code>expect</sky-code> function into spec files, the next step is to wrap the <sky-code>expect</sky-code> function around the elements to test for accessibility. Accessibility checks are asynchronous, so it's important to wrap your tests in Angular's <sky-code>async</sky-code> method. And to break the linear flow of control when using <sky-code>async</sky-code>, it's also important to wrap <sky-code>toBeAccessible</sky-code> in Angular's <sky-code>whenStable</sky-code> method.
+    After you import the <sky-code>expectAsync</sky-code> function into spec files, the next step is to wrap the <sky-code>expectAsync</sky-code> function around the elements to test for accessibility.
   </p>
 
   <sky-code-block>
-    it('should pass accessibility', async(() => {
-      fixture.whenStable().then(() => {
-        expect(element).toBeAccessible();
-      });
+    it('should pass accessibility', async () => {
+      await expectAsync(element).toBeAccessible();
     }));
   </sky-code-block>
 
-  <p>
-    Keep in mind that <sky-code>async</sky-code> is also a native JavaScript declaration, so you need to explicitly import the Angular <sky-code>async</sky-code> method at the top of the unit test spec file.
-  </p>
-
-  <sky-code-block>
-    import { async } from ‘@angular/core/testing’;
-  </sky-code-block>
-
   <stache-page-anchor>
     Specify parameters for <sky-code>toBeAccessible</sky-code>
   </stache-page-anchor>
 
   <p>
-    Finally, you can specify two optional parameters within the <sky-code>toBeAccessible</sky-code> matcher. The first parameter is a callback that lets you execute code after accessibility checks run. The second parameter is an option that lets you overwrite <a href="https://github.com/blackbaud/skyux-sdk-testing/blob/master/src/app/public/a11y/a11y-analyzer.ts">the default accessibility checks that <sky-code>SkyA11yAnalyzerConfig</sky-code> specifies</a>.
-  </p>
-
-  <h3 class="sky-subsection-heading">
-    Callback
-  </h3>
-  <p>
-    In the first <sky-code>toBeAccessible</sky-code> parameter, you can specify logic to run after accessibility testing:
-  </p>
-  <sky-code-block>
-    it('should pass accessibility', async(() => {
-      fixture.whenStable().then(() => {
-        expect(element).toBeAccessible(() => {
-          // Do something else after the accessibility checks run...
-        });
-      });
-    }));
-  </sky-code-block>
-  <p>
-    For example, for components that change state, you can use this parameter to change state after accessibility checks and then run the accessibility checks again.
+    Finally, you can specify an optional parameter within the <sky-code>toBeAccessible</sky-code> matcher that lets you overwrite <a href="https://github.com/blackbaud/skyux-sdk-testing/blob/master/src/app/public/a11y/a11y-analyzer.ts">the default accessibility checks that <sky-code>SkyA11yAnalyzerConfig</sky-code> specifies</a>.
   </p>
 
   <h3 class="sky-subsection-heading">
```

Comment:
As mentioned above, I'd delete this header now.

Reply:
Done!